admin/package.json

```diff
@@ -44,7 +44,7 @@
     "keycloak-angular": "5.1.0",
     "leaflet": "1.5.1",
     "linkify-it": "2.2.0",
-    "lodash": "4.17.15",
+    "lodash": "4.17.13",
```

Comment:
Remove lodash

Reply:
This might be better as its own jira task, since lodash is used in the code in about 100 places currently.

Reply:
Created a ticket: https://bcmines.atlassian.net/browse/PTI-140